libgnucash/engine/Account.h

```diff
@@ -1542,6 +1564,7 @@ const char * dxaccAccountGetQuoteTZ (const Account *account);
 #define ACCOUNT_NOTES_		"notes"
 #define ACCOUNT_BALANCE_	"balance"
 #define ACCOUNT_NOCLOSING_	"noclosing"
+#define ACCOUNT_OPENING_BALANCE "opening-balance"
```

Comment:
`ACCOUNT_IS_OPENING_BALANCE_`

Dunno why the trailing underscore, but be consistent.